apps/website/package.json

```diff
@@ -5,7 +5,6 @@
   "keywords": [],
   "homepage": "http://devlaunchers.org/",
   "dependencies": {
-    "@cloudflare/wrangler": "^1.19.12",
```

Comment:
@chungthuang I removed `@cloudflare/wrangler` from the dependencies because it's deprecated and makes my github action fails. is this not used anywhere else?